### PR TITLE
Add pyepics package

### PR DESCRIPTION
**Pull request by marcomontevechi1**
As part of UNICAMP's project with professor @hyviquel

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and version sha256.  Adding @adamjstewart since he's the python package maintainer.

File: var/spack/repos/builtin/packages/py-epics/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyEpics(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-epics/package.py:10]: Package name should be `py-pyepics`

File: var/spack/repos/builtin/packages/py-epics/package.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    """EPICS channel access for python. A thin layer of abstraction
+    for libca with some tools built above it.
+
+    You can learn more about EPICS here: https://epics-controls.org/resources-and-support/documents/getting-started/
+    """
+
+    homepage = "http://pyepics.github.io/pyepics/"
+    url = "https://files.pythonhosted.org/packages/6c/cd/f97b676b6903e6f88bfced3817964559c20a1e3c6cd5244fa188f76fff75/pyepics-3.5.1.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-epics/package.py:18]: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#id2

File: var/spack/repos/builtin/packages/py-epics/package.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    maintainers("MarcoMontevechiFilho1")
+
+    version("3.5.1", sha256="a4d0f2d0d163aa34a53f560519f5664a42ba96aeb19bbf92e46228f22fa87ff6")
+
+    depends_on("py-setuptools", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-epics/package.py:24]:
```suggestion
    depends_on("py-setuptools", type=("build", "run"))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by alecbcs** [comment]
Going to close this PR due to a lack of response after review feedback. @marcomontevechi1 happy to reopen if you still want to get this merged and have time to work on it.

**Pull request closed without merging**